test/hotspot/gtest/aarch64/aarch64-asmtest.py

```diff
@@ -1396,6 +1457,10 @@ def generate(kind, names):
                      "autiza", "autizb", "autdza", "autdzb", "xpacd",
                      "braaz", "brabz", "blraaz", "blrabz"])
 
+for system_reg in ["fpsr", "dczid_el0", "ctr_el0", "nzcv"]:
+    generate (SystemOneRegOp, [ ["msr", system_reg] ])
+    generate (OneRegSystemOp, [ ["mrs", system_reg] ])
```

Comment:
```suggestion
for system_reg in ["fpsr", "nzcv"]:
    generate (SystemOneRegOp, [ ["msr", system_reg] ])
 
for system_reg in ["fpsr", "dczid_el0", "ctr_el0", "nzcv"]:
    generate (OneRegSystemOp, [ ["mrs", system_reg] ])
```

1. `asmtest.out.h` should be updated as well. Note that it's generated by running `aarch64-asmtest.py` script.
2. There are no instructions to "set/update" `dczid_el0` or `ctr_el0` in `macroAssembler_aarch64.hpp`. Here shows the warnings of running `python2 aarch64-asmtest.py`:
```
aarch64ops.s: Assembler messages:
aarch64ops.s:202: Warning: specified register cannot be written to at operand 1 -- `msr dczid_el0,x27'
aarch64ops.s:206: Warning: specified register cannot be written to at operand 1 -- `msr ctr_el0,x11'
```

Reply:
Oops. I fixed the test and added the generated code as suggested.